### ISSUE TITLE
Street address line 2 does not have a field name (fails accessibility requirements)

### ISSUE THREAD

**Issue by ziranshang** [google/libaddressinput#228]
Street address line 2 does not have a field name: https://github.com/google/libaddressinput/blob/master/android/src/main/java/com/android/i18n/addressinput/AddressWidget.java#L388

 This means it does not have a hint or label, so it fails accessibility testing.

**Comment by jmealo** [comment]
This would be very useful! :+1:
